### ISSUE TITLE
Acquire a signing service opens up Comet

### ISSUE THREAD

**Issue by KennethAshley** [vechain/connex#90]
Comet does not support Connex 2 but acquiring a singing service opens up Connex. It should open up Sync2

**Comment by qianbin** [comment]
It was designed to. The connex2 package can work in compatible-mode when connex1 environment detected, to make dapps built with connex2 run properly in sync1 or with comet plugin.

**Comment by KennethAshley** [comment]
But executing a transaction doesn't work with Sync 1 or Comet. Our users have been having problems

**Comment by qianbin** [comment]
> But executing a transaction doesn't work with Sync 1 or Comet. Our users have been having problems

ok. I think we need an option to disable the feature.

btw. is there any detail of users' problems?

**Comment by KennethAshley** [comment]
What happens is users who have Comet installed are asked to unlock their account so they think Comet works but once they unlock their account Connex 2 takes over and the site breaks

**Comment by qianbin** [comment]
> What happens is users who have Comet installed are asked to unlock their account so they think Comet works but once they unlock their account Connex 2 takes over and the site breaks

I just published v2.0.5 with the new option `noV1Compat`. The example is like this:

```typescript
const connex = new Connex({
    node: 'some what',
    network: 'main',
    noV1Compat: true
})
```

**Comment by KennethAshley** [comment]
That was quick! Thanks a bunch!
